### DISCUSSION TITLE
usb connection disappeared after flashing

### DISCUSSION THREAD

**Discussion by danny1209** [OE4T/tegra-demo-distro#209] [Q&A] [answered]
Hi, 

I´m using "Jeson nano module(emmc) + Carrier board(Leopard Image)"  the board can be flashed successfully, but after flashing the usb connection is disappeared. In my hostmachine I cannot see the removable usb connection any more.  Does somebody have any idea?  

Thanks in advance 
Danny

**Comment by madisongh** [comment, marked as answer]
I'm assuming you're talking about having the Jetson show up as a device on your development host.  We don't include the L4T device mode support in the demo images.  You can add it with
```
CORE_IMAGE_EXTRA_INSTALL = "l4t-usb-device-mode"
```
in your local.conf file.

It works a bit differently than what NVIDIA provides in their Ubuntu images, but the network and serial support are there.  See [this wiki page](https://github.com/OE4T/meta-tegra/wiki/USB-device-mode-support) for more information.   It also doesn't get much testing, but the last time I checked, it worked.

**Reply by danny1209** [reply]
Hi @madisongh ,

yes, exactly! it also works for me. thanks a lot!
I see this package is already in the repository "meta-tegra". Is that repository include more packages than this one. which one is recommended to use for development?

**Reply by madisongh** [reply]
> which one is recommended to use for development?

There's no single answer for this... it really depends on what you're developing and what JetPack features you need.

**Reply by danny1209** [reply]
I´m developing the driver and application for a camera module, we need gstreamer, v4l2, cuda ...
